### PR TITLE
Added telemetry flag

### PR DESCRIPTION
**Pull request by MrMCake**
# Description

- Added telemetry flag

| Pipeline |
| - |
|[AnalysisServices: Servers](https://github.com/Azure/ResourceModules/actions/workflows/ms.analysisservices.servers.yml) |

# Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (Wiki)

### CODE REVIEW

File: constructs/Microsoft.Authorization/roleAssignments-multiRolesMultiPrincipals/readme.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 This module deploys Role Assignments.
 
+## Navigation
```
Comment (eriqua) [inline constructs/Microsoft.Authorization/roleAssignments-multiRolesMultiPrincipals/readme.md:5]: Do we need module readmes in the costructs folder?

Comment (MrMCake) [reply constructs/Microsoft.Authorization/roleAssignments-multiRolesMultiPrincipals/readme.md:5]: @eriqua I think we do. But regardless it's something we can discuss outside of this PR. (I just re-ran the generation to make sure all modules compile)

File: arm/Microsoft.Network/virtualNetworks/deploy.bicep [hunk 2]
```diff
@@ -186,7 +187,7 @@ module virtualNetwork_peering_local 'virtualNetworkPeerings/deploy.bicep' = [for
     allowGatewayTransit: contains(peering, 'allowGatewayTransit') ? peering.allowGatewayTransit : false
     allowVirtualNetworkAccess: contains(peering, 'allowVirtualNetworkAccess') ? peering.allowVirtualNetworkAccess : true
     doNotVerifyRemoteGateways: contains(peering, 'doNotVerifyRemoteGateways') ? peering.doNotVerifyRemoteGateways : true
-    useRemoteGateways: contains(peering, 'useRemoteGateways') ? peering.useRemoteGateways : false
```
[the remark this answers is not in the document]
Comment (MrMCake) [reply arm/Microsoft.Network/virtualNetworks/deploy.bicep:189]: nope

File: arm/Microsoft.Network/virtualNetworks/deploy.bicep [hunk 3]
```diff
@@ -186,7 +187,7 @@ module virtualNetwork_peering_local 'virtualNetworkPeerings/deploy.bicep' = [for
...
+    enableDefaultTelemetry: enableDefaultTelemetry
```
Comment (MrMCake) [inline arm/Microsoft.Network/virtualNetworks/deploy.bicep:190]:
```suggestion
    useRemoteGateways: contains(peering, 'useRemoteGateways') ? peering.useRemoteGateways : false
    enableDefaultTelemetry: enableDefaultTelemetry
```

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**